core/primitives/src/views.rs

```diff
@@ -328,6 +328,7 @@ pub struct BlockHeaderView {
     pub outcome_root: CryptoHash,
     pub chunks_included: u64,
     pub challenges_root: CryptoHash,
+    #[serde(with = "u64_dec_format")]
```

Comment:
@frol should I create a separate field `real_timestamp` for backward compatibility?